packages/terra-list/src/List.jsx

```diff
@@ -77,6 +78,22 @@ const propTypes = {
    * One of `'none'`, `'single-select'`, `'multi-select'`.
    */
   ariaSelectionStyle: PropTypes.oneOf(['none', 'single-select', 'multi-select']),
+  /**
+   * Whether or not the list item is draggable. List Item is draggable only when it is selectable.
+   */
+  isDraggable: PropTypes.bool,
+  /**
+   * Function callback when the Item is Lifted.
+   */
+  onDragStart: PropTypes.func,
+  /**
+   * Function callback when the Item is dragged from one position to another.
+   */
+  onDragUpdate: PropTypes.func,
+  /**
+   * Function callback when the Item is dropped.
+   */
+  onDragEnd: PropTypes.func,
```

Comment:
Are there any parameters for these callback functions?

Reply:
The official docs of react-beautiful dnd says to block any updates `onDragStart` and `onDragUpdate`. So lets remove these 2 props and keep only onDragEnd. onDragEnd takes 2 parameters `result`, `provided`.

Reply:
Changes Done Thanks.